utils/download_glue_data.py

```diff
@@ -0,0 +1,144 @@
+''' Script for downloading all GLUE data.
+
+Original source: https://github.com/kamalkraj/ALBERT-TF2.0/blob/fa90194e5fe729dbb19f32ac29c8d6d6372c0f93/download_glue_data.py
+Original license: https://github.com/kamalkraj/ALBERT-TF2.0/blob/fa90194e5fe729dbb19f32ac29c8d6d6372c0f93/LICENSE (Apache-2.0)
```

Comment:
That's not the original source, this script was made by NYU.
It should probably be: https://gist.github.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e

Reply:
I changed the link to what you suggested, but not sure what is the copyright or the license of a gist? Maybe just the link to the gist is enough?